### PR TITLE
Activity Indicator When logging in

### PR DESCRIPTION
**Pull request by sivaram16**
Activity Indicator When logging in

Issue : https://github.com/nordenmark/yaniv/issues/7

### CODE REVIEW

File: lib/screens/login.screen.dart [hunk 1]
```diff
@@ -25,20 +40,20 @@ class LoginState extends State<LoginScreen> {
...
+      print("user" + user.toString());
+
+      setState(() {
+        showIndicator = false;
+      });
 
       firebaseService.setEmail(user.email);
       Navigator.pushNamed(context, '/games');
     } catch (e) {
+      print("fsf");
```
Comment (damoonrashidi) [inline lib/screens/login.screen.dart]: Left over debugging message?

File: lib/screens/login.screen.dart [hunk 2]
```diff
@@ -13,8 +14,22 @@ class LoginState extends State<LoginScreen> {
   final GoogleSignIn _googleSignIn = GoogleSignIn(scopes: ['email']);
   final FirebaseAuth _auth = FirebaseAuth.instance;
   final FirebaseService firebaseService = FirebaseService();
+  bool showIndicator = false; 
 
+  @override
+  build(BuildContext context) => Scaffold(
+        body: showIndicator
+            ? Center(child: CupertinoActivityIndicator())
```
Comment (damoonrashidi) [inline lib/screens/login.screen.dart]: Preferably this should be a`CircularProgressIndicator` or similar instead of the Cupertino one.

**Review by damoonrashidi — CHANGES_REQUESTED** [review]
Great job, just two small comments! Thanks for the PR

**Comment by sivaram16** [comment]
Now changed Go ahead!

**Review by damoonrashidi — APPROVED** [review]
Thanks! looks good to me!

**Pull request merged**